glib/src/value.rs

```diff
@@ -21,15 +17,14 @@
 //!
 //! ```
 //! use glib::prelude::*; // or `use gtk::prelude::*;`
-//! use glib::{TypedValue, Value};
+//! use glib::Value;
 //!
-//! // Value and TypedValue implement From<&i32>, From<&str>
-//! // and From<Option<&str>>. Another option is the `ToValue` trait.
+//! // Value implement From<&i32>, From<&str> // and From<Option<&str>>.
```

Comment:
Spurious `//` in the middle of the line?

Reply:
Thanks!